docs/configuration/config-file-reference.md

```diff
@@ -3550,6 +3550,9 @@ The `limits_config` configures default and per-tenant limits imposed by Cortex s
 # CLI flag: -frontend.max-queriers-per-tenant
 [max_queriers_per_tenant: <float> | default = 0]
 
+# Enable query to return partial data with warning message.
```

Comment:
I'm assuming this defaults to getting response from at least 1 ingester. 

should this also be configurable and always needs to be lower than `replication_factor`? would a user want 2 out of 5 replica instead of 1 out of 5?

I think at least 1 is a good start. if the need emerges, we can re-consider

Reply:
I agree. The base assumption of this feature is that zonal replication is reliable, so I wanted to keep the config nice and simple. We can definitely re-iterate in the future if needed.